lua/gen/init.lua

```diff
@@ -253,7 +262,7 @@ M.exec = function(options)
             end
         end,
         on_stderr = function(_, data, _)
-            if opts.debug then
+            if opts.debug and false then
```

Comment:
```suggestion
            if opts.debug then
```

Reply:
I commented disabled debug, since I remember it was not helping in debugging the request. Cant remember exactly what was the issue. But I restored it in 93c5eadcdb19bfea21cf2767baa5a4345532f91f